Documentation/cli-installation-scenarios.md

```diff
@@ -0,0 +1,241 @@
+  * [Visual Studio](#visual-studio)
+  * [VS Code](#vs-code)
+  
+
+# Overview
+This document/spec outlines the CLI install experience. This covers the technologies being used for install, the principles that are driving the installation experience, the ways users are coming to the installs and what each of the installs contains, in terms of stability and similar. 
+
+# General principles
+
+- All installers are signed properly 
+- Upgrades using the native installers Just Work(tm)
+- All user facing materials point to the getting started page
+- The user needs extra effort to install the "bleeding edge" bits
+- Only HTTPS links are allowed in any online property 
+
+# Channels
+Channels represent a way for users who are getting the bits to reason about the stability and quality of the bits they are getting. Each channel has, potentially, a different 
+
+The table below outlines the channels:
+
+| Property         	| Description                                                                                                                                                                                                                                                       	|
+|------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| Dev              	| Unstable bits that are "bleeding edge". Users are nto expected to us this channel often, however it is there for those situations when someone needs a feature or maybe a new bug fix urgently that hasn't been stabilizied yet. Also, used for internal testing. 	|
+| Beta/Pre-release 	| Pre-release stable builds with known issues and/or known feature gaps. We are OK with users installing these for preview purposes.                                                                                                                                	|
+| Release          	| Actual releases. Most users are encouraged to install these.                                                                                                                                                                                                      	|
+Channels also impact the NuGet packages. Based on the version of the package specified, the user may end up getting different channels. This is described in more details in [Nuget Packages section](#nuget-packages).  
+
+# Dependencies
+.NET Core CLI is built on top of CoreFX and CoreCLR and as such its' dependencies set is defined by the platform that those two combine. Some of the install options provide an automatic way to install these dependencies, while other do not. However, it is important to note that the CLI bundle will **not carry those dependencies with it** in any case. This means that for those install options that do not support automatic 
```

Comment:
Whether the CLI will carry the runtime and how that runtime will be used is still TBD (the current state is that it is carrying it). However, this applies on both cases I believe since CLI bundle will not install those dependencies and will not carry it with them.